### PR TITLE
A11y workshop/landing accessible

### PR DESCRIPTION
**Pull request by alandours**
* Add HomePageAccessible `/accesible` with a link to a contact form `/contacto`
Screen Shot 2022-07-21 at 11 03 05

* Add ContactPage `/contacto` with the contact form
Screen Shot 2022-07-21 at 11 03 18

### CODE REVIEW

File: food-app/src/pages/ContactPage.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+export const ContactPage = () => {
+  const [sent, setSent] = useState(false);
+
+  return (
+    <section className="container px-4 mx-auto">
+      <div className="flex flex-col">
+        <h1 className="text-3xl font-extrabold mb-5">Envianos un mensaje</h1>
+        {sent ? (
+          <div>Tu mensaje fue enviado, te responderemos pronto.</div>
```
Comment (aguscha333) [inline food-app/src/pages/ContactPage.js:11]:
```suggestion
          <div role="alert">Tu mensaje fue enviado, te responderemos pronto.</div>
```

> Sometimes, for example, when using AJAX techniques, the browser is not loading a new page but shows changes, such as form errors, dynamically on the page. The list of errors should be inserted into a prominent container on the top to inform the user in such a case. In addition to the advice above, this container should have the role attribute set to alert to make assistive technology users aware of this change.

taken from [this W3C page in the Listing Errors section](https://www.w3.org/WAI/tutorials/forms/notifications/)

File: food-app/src/pages/ContactPage.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            <label for="message" className="mt-5 mb-1">
+              Mensaje
+            </label>
+            <textarea
+              id="message"
+              className="p-1 h-48 rounded bg-gray-50 border border-slate-300 resize-none"
+              required
+            />
+            <button type="submit" className="bg-orange-400 rounded p-2 mt-5">
+              Envíar mensaje
```
Comment (aguscha333) [inline food-app/src/pages/ContactPage.js:43]:
```suggestion
              Enviar mensaje
```

**Review by aguscha333 — COMMENTED** [review]
great work

**Review by agustinmrgo — APPROVED** [review]

**Review by elizabethlofredo — APPROVED** [review]
Nice job 🙌 I didn't pull the branch to test it but code changes look good to me

**Pull request merged**